src/popup/components/InputAccount.vue

```diff
@@ -0,0 +1,531 @@
+<!-- eslint-disable vuejs-accessibility/label-has-for -->
```

Comment:
This whole fail is a copy & paste of an `InputField` with a one element `dropdown-wrapper`. Is it possible to achieve same result without copying 443 lines of code of initial `InputField`?

Reply:
It has more changes than just the `dropdown-wrapper` element. Initially I tried re-using the InputField component but it was harder to maintain because of all the extra conditions. I believe it's much cleaner in separate components, but I can retry to make it cleaner in a single component.

Reply:
Copy + paste is the worse way of solving issues and I 100% agree with Nikita. `InputField` component is a base component which is used in many other input components by extending it. I suggest to follow this pattern.